nixos/modules/services/admin/oddjobd.nix

```diff
@@ -0,0 +1,36 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  pkg = pkgs.oddjob;
```

Comment:
A configurable `package` option would be nice.

Reply:
What other package would be used? Or do you mean for overrides?

Reply:
Just so people can easily use overrides. Most other modules have an `options.*.package` option.